### PR TITLE
feat/new_receiver

### PR DESCRIPTION
**Pull request by xross**
Merge new receiver into develop

### CODE REVIEW

**Comment by xross** [comment]
[lib_spdif.pdf](https://github.com/xmos/lib_spdif/files/13001391/lib_spdif.pdf)

**Comment by xross** [comment]
PDF document:

[lib_spdif.pdf](https://github.com/xmos/lib_spdif/files/13002979/lib_spdif.pdf)

**Comment by xross** [comment]
Updated doc: 
[lib_spdif.pdf](https://github.com/xmos/lib_spdif/files/13008118/lib_spdif.pdf)

**Review by danielpieczko — APPROVED** [review]

File: lib_spdif/src/SpdifReceive.xc [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    for(int i=0; i<5000;i++) // 5000 32 bit samples @ 100MHz takes 1.6ms
+    {
+        p :> sample;
+        if (cls(sample) > max_pulse)
+        {
+            max_pulse = cls(sample);
+        }
+    }
+
+    // Now find the minimum pulse width
```
[the remark this answers is not in the document]
Comment (xross) [reply lib_spdif/src/SpdifReceive.xc]: Fixed

File: README.rst [hunk 2]
```diff
@@ -1,43 +1,31 @@
-S/PDIF library
-##############
+S/PDIF Component
+################
 
 :Version: 5.0.1
```
[the remark this answers is not in the document]
Comment (xross) [reply README.rst:4]: This will be update by the release script

**Pull request merged**